iotedgedev/modules.py

```diff
@@ -128,9 +128,18 @@ def build_push(self, no_build=False, no_push=False):
                         if not no_push:
                             # PUSH TO CONTAINER REGISTRY
                             self.output.info("PUSHING DOCKER IMAGE TO: " + image_destination_name)
+                            # FIND ENV VAR KEY BASED ON REGISTRY VALUE
+                            registry_key = None
+                            for registry in self.envvars.CONTAINER_REGISTRY:
+                                if self.envvars.CONTAINER_REGISTRY[registry].server == module_json.registry:
+                                    registry_key = registry
+                                    break
+                            if registry_key is None:
+                                self.output.error("Error while trying to retrieve container registry information.")
```

Comment:
How about "Could not find registry server with key {0}. Please make sure your envvar is set.